src/components/UserList/Row.js

```diff
@@ -162,6 +163,33 @@ class Row extends Component {
       },
     ];
 
+    let connectButton = null;
+
+    if (this.state.pending) {
+      connectButton = (
+        <View style={[styles.connectBtnStyle, { backgroundColor: '#a6aeae' }]}>
+          <View style={styles.buttonContainer}>
+            <View style={{ paddingTop: 10, marginRight: 5, }}>
+              <Icon name="clock-o" size={15} color="white" />
+            </View>
+            <Text style={styles.buttonText}>WAITING</Text>
+          </View>
+        </View>
+      );
+    } else {
+      connectButton = (
+        <LinearGradient style={styles.connectBtnStyle} start={[0.9, 0.5]} end={[0.0, 0.5]}
+                        locations={[0, 0.75]}
```

Comment:
Expected indentation of 10 space characters but found 24

Next line(183) is same.